src/style/reset.scss

```diff
@@ -0,0 +1,12 @@
+button, a, img, input, select, textarea {
+  -webkit-tap-highlight-color: rgba(0,0,0,0);
+}
+
+a, button, img, [inert], .inert {
+  user-select: none;
+  -webkit-user-select: none;
+  user-drag: none;
+  -webkit-user-drag: none;
```

Comment:
Should we use something like [autoprefixer](https://github.com/postcss/autoprefixer)?

Reply:
I don't think we should decide that as part of this PR.